package.json

```diff
@@ -1,26 +1,27 @@
 {
   "name": "tedious-newrelic",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "description": "Instrumentation for tedious node lib",
   "main": "index.js",
   "engines": {
-    "node": ">=6.0.0 <=12.0.0",
+    "node": ">=6.0.0 <13.0.0",
```

Comment:
This is no net change.  It went from `<=12` to `<13`.

Reply:
Because it's `12.0.0` and not `12` I think it does pose a problem.
Example:
```
WARN notsup Unsupported engine for tedious-newrelic@0.3.0: wanted: {"node":">=6.0.0 <=12.0.0","npm":">=2.0.0"} (current: {"node":"12.13.0","npm":"6.12.0"})
npm WARN notsup Not compatible with your version of node/npm: tedious-newrelic@0.3.0
```